src/pages/index.js

```diff
@@ -54,9 +55,30 @@ const IndexPage = (props) => {
                         </Box>
                     </Box>
                 </Link>
-            )
+            );
         }
-    })
+    });
+
+    const [thankYou, setThankYou] = React.useState([]);
+
+    useEffect(() => {
+        const interval = setInterval(() => {
+            axios
+                .get(
+                    'https://raw.githubusercontent.com/jmMeessen/jenkins-submitter-stats/main/data/honored_contributor.csv',
```

Comment:
Runtime fetching of content not in Jenkins infra org seems super risky.

Reply:
Also why csv?

Reply:
Later this month, or during the summer, the [repo will be migrated](https://github.com/jenkins-infra/helpdesk/issues/4017) to the jenkins infra organization.

Reply:
as an dev that does a lot of maintenance work, nothing says it won't get done like saying "it'll get done later"
I'm not going to try and block it of course, but I'd say moving it is a higher priority than using it.

Reply:
![image](https://github.com/jenkins-infra/contributor-spotlight/assets/110087/5ac130ac-8a32-4e3f-84f4-0646391d0ce4)
kinda proves my point :)

Reply:
Repositories imported.